src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -331,7 +483,11 @@ def _merge(self, dict_1, dict_2, identifier):
                 merged_dict[key] = list(merged_dict[key] + [value])
             elif isinstance(value, list):
                 cleaned_list = list(filter(None, chain(value, merged_dict[key])))
-                merged_dict[key] = list(set(cleaned_list))
+                if len(cleaned_list) > 0 and isinstance(cleaned_list[0], dict):  # make dicts hashable
+                    hashable_dicts = [freeze(d) for d in cleaned_list]
```

Comment:
I think `hashable_dicts = freeze(cleaned_list)` should also work but the comment would have to change to `Make each dict in cleaned_list hashable`